README.md

```diff
@@ -195,12 +195,16 @@ Make sure you have the `nightly` toolchain installed locally:
 
 ```bash
 rustup toolchain install nightly
+cargo install --force cbindgen
+rustup target add wasm32-unknown-unknown
 ```
 
 After all the requirements are met, you can then run:
 
 ```bash
-cd parser
+cd parser/tools
```

Comment:
Do you mind moving all commands in the first block and leave the last one alone?
The rest LGTM! :)